azure-pipelines.yml

```diff
@@ -42,280 +33,239 @@ resources:
+            # Use utility script to run script command dependent on agent OS.
+            - script: eng\common\cibuild.cmd
                 -configuration $(_BuildConfig)
-                -restore
                 -prepareMachine
-                -sign
-                -ci
-                /p:DotNetSignType=$(_SignType)
-                /p:TeamName=DotNetCore
-                /p:OfficialBuildId=$(BUILD.BUILDNUMBER)
-              displayName: Sign packages
+                $(_InternalBuildArgs)
+              displayName: Windows Build / Publish
+            - task: ComponentGovernanceComponentDetection@0
+              displayName: Component Governance scan
 
-- ${{ if and(ne(variables['System.TeamProject'], 'public'), notin(variables['Build.Reason'], 'PullRequest')) }}:
-  - stage: Create_BAR_ID_Tag
-    displayName: Create BAR ID Tag
-    condition: succeededOrFailed()
-    jobs:
-    - template: /eng/common/templates/job/job.yml
-      parameters:
-        name: Create_BAR_ID_Tag
-        displayName: Create BAR ID Tag
-        pool: 
-          name: $(PoolProvider)
-          demands: ImageOverride -equals windows.vs2019.amd64
-        variables:
-          - group: Publish-Build-Assets
-          - group: DotNetBot-GitHub
-        steps:
-          - checkout: self
-            clean: true
-          - powershell: eng/create-baridtag.ps1
-              -azdoToken $(dn-bot-dotnet-build-rw-code-rw)
-              -barToken $(MaestroAccessToken)
-              -githubPAT $(BotAccount-dotnet-bot-repo-PAT)
-            displayName: Create BAR ID Tag
-            name: Create_BAR_ID_Tag
-  - template: eng\common\templates\post-build\post-build.yml
-    parameters:
-      publishingInfraVersion: 3
-      # Symbol validation isn't being very reliable lately. This should be enabled back
-      # once this issue is resolved: https://github.com/dotnet/arcade/issues/2871
-      enableSymbolValidation: false
-      enableSourceLinkValidation: true
-      # This is to enable SDL runs part of Post-Build Validation Stage
-      SDLValidationParameters:
-        enable: true
-        params: ' -SourceToolsList @("policheck","credscan")
-        -TsaInstanceURL $(_TsaInstanceURL)
-        -TsaProjectName $(_TsaProjectName)
-        -TsaNotificationEmail $(_TsaNotificationEmail)
-        -TsaCodebaseAdmin $(_TsaCodebaseAdmin)
-        -TsaBugAreaPath $(_TsaBugAreaPath)
-        -TsaIterationPath $(_TsaIterationPath)
-        -TsaRepositoryName "Arcade-Validation"
-        -TsaCodebaseName "Arcade-Validation"
-        -TsaPublish $True'
-  - ${{ if eq(variables['Build.SourceBranch'], 'refs/heads/release/8.0') }}:
-    - stage: Validate_Publishing
-      displayName: Validate Publishing
+          - job: Linux
+            container: LinuxContainer
+            pool:
+              name: $(PoolProvider)
+              image: 1es-ubuntu-2204
+              os: linux
+            strategy:
+              matrix:
```

Comment:
this and the last matrix strategy confuse me. why are we testing debug and test signing in official builds❔

Reply:
@riarenas do you know if this is for the scheduled part? or can we remove it (this and the one in line 153)

Reply:
Since arcade-validation tests all the functioning of Arcade, we exercise test signing in internal, and debug in internal to make sure there's no regressions.